client/themes/bcgov/assets/i18n/en.json

```diff
@@ -25,7 +25,12 @@
     "demo.person_relationship.registries.ca": "Relationship",
     "demo.org_relationship.registries.ca": "Relationship",
     "Cannabis Retail Store Licence": "Cannabis Retail Store Licence",
-    "Cannabis Marketing Licence": "Cannabis Marketing Licence"
+    "Cannabis Marketing Licence": "Cannabis Marketing Licence",
+    "business_number": "Business Number",
+    "source_id": "BC Registries ID",
+    "display_name": "Name",
+    "entity_name": "Name",
+    "entity_name_assumed": "Name"
```

Comment:
Should we differentiate at all when displaying which name we are using? Nothing wrong with only having `Name`, wondering about making explicit that it is `Name`, `Name (Entity)` or `Name (Assumed)` or something like that.

Reply:
For OrgBook, we should be displaying the name that the business is operating under in BC, which will be their legal name.

For most companies it will be "entity_name".

For extra-provincial companies it may be "entity_name_assumed" (they registered under "entity_name" somewhere else but then couldn't use this name in BC, so they had to "assume" a different name.)

If there is *no name available* (hopefully this won't happen in `prod`) then we can use "source_id".  (This is the BC-specific company number.)

I understand that eventually the BN will also be a "name", but should never be displayed as the company "name" in OrgBook.  (It's a federal number and it will be misleading to display this as the "name".)

Reply:
Under the current implementation, unless the credential issuer specifically maps the BN to a `display_name`, `entity_name` or `entity_assumed_name` name type, it will not be shown as the main name. It will instead display in list of alternate identifiers. For my own test cases, BN's are still using `business_number` as the name type so that the labels are formatted correctly.

Reply:
> Should we differentiate at all when displaying which name we are using?
> 
> Nothing wrong with only having `Name`, wondering whether we care about making it explicit that it is `Name`, `Name (Entity)` or `Name (Assumed)` or something like that.

I would be ok with updating the labels to correctly reflect what they are. In the case of OrgBook, the name type doesn't get displayed in the search result titles, so perhaps we could add some logic there to add the type in parentheses if it is an assumed name?

Reply:
I am lost as we got into naming logic details I am not familiar with.

My point was that the labels associated with those types are the same and there is no way of knowing which one of those was used at display time. If Aries VCR can display all of them, maybe we should use labels that are indicative of each one.

I agree that additional, BC-specific logic would be incorporated into the BCGov theme, - the assets file belongs to the BCGov theme, so I assumed those types were used, but it appears they are not? We may want to connect in person to clarify.

Reply:
@esune -- at an Aries VCR level, we want to enable topics to have multiple names and to be able to say "this is the name we display and the rest are alternatives". That gives us a way to ensure we always show the same name regardless of how we found the topic.

BTW - this conversation tells me that a singleton "displayname" is not enough -- we need an ordered list to display the name we have -- in case the priority one is not in the database.

Reply:
Ok, I understand that. It sounds like there are two parts to this conversation then:
1. The labels I marked should be more explicit, so we know which name we are displaying with the current fallback logic.
2. The display logic that is currently baked-in the BCGov component (if I understand/remember correctly) may need to be reviewed and extended to be generalized and - perhaps - configurable (the order) in any Aries VCR instance.